pkg/model/response/response.go

```diff
@@ -0,0 +1,399 @@
+// Package response contains plain old Go types returned by wharf-web in the
+// HTTP responses, with Swaggo-specific Go tags.
+//
+// Copied from github.com/iver-wharf/wharf-api/pkg/model/response
```

Comment:
nit: Would be nice with a version reference. Example:

```suggestion
// Copied from https://github.com/iver-wharf/wharf-api/blob/v5.0.0/pkg/model/response/response.go
```